app/test_main.py

```diff
@@ -1,3 +1,22 @@
 from app.main import get_human_age
+import pytest
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "init_cat_age,init_dog_age,expected_result",
+    [
+        pytest.param(24, 24, [2, 2], id="24/24 cat/dog age"),
+        pytest.param(14, 14, [0, 0], id="14/14 cat/dog age"),
+        pytest.param(15, 15, [1, 1], id="15/15 cat/dog age"),
+        pytest.param(23, 23, [1, 1], id="23/23 cat/dog age"),
+        pytest.param(27, 28, [2, 2], id="27/28 cat/dog age"),
+        pytest.param(28, 29, [3, 3], id="28/29 cat/dog age"),
+    ]
```

Comment:
Consider adding test cases for edge cases such as negative numbers, zero, or very large numbers to ensure the function handles all possible inputs robustly. This is important to meet the checklist requirements.